client/components/site-selector/index.jsx

```diff
@@ -310,6 +316,14 @@ export class SiteSelector extends Component {
 			sites = sites.filter( ( site ) => site.slug !== this.props.selected );
 		}
 
+		// Previously, each domain-only site would correspond to one site in the list.
+		// Soon, bulk transfers of many domains will be attached to a single domain-only site.
+		// Because of this, it no longer makes sense to show domain-only sites in the
+		// site selector.
```

Comment:
Seems like this comment will go out of date pretty quickly. I don't know how far away bulk-domain transfers are, but works already begun maybe it'd be better to just comment in the present tense.

```suggestion
		// Bulk transfers of many domains get attached to a single domain-only site.
		// Because of this, it doesn't make sense to show domain-only sites in the site selector.
```

Reply:
Good suggestion -- making the changes 🙂